build-tools/fetch-schemas-plugin.ts

```diff
@@ -1,39 +1,40 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2024 Pionix GmbH and Contributors to EVerest
-import {remoteSchemas} from "../schemas.config";
-import yaml from 'js-yaml';
-import crypto from 'crypto';
+import { remoteSchemas, } from "../schemas.config";
+import yaml from "js-yaml";
+import crypto from "crypto";
 import fs from "fs";
 
 
 export function vitePluginFetchSchemas() {
-    return {
-        name: 'vite-plugin-fetch-schema',
-        buildStart() {
-            fetchRemoteSchemas();
-        }
-    };
+  return {
+    name: "vite-plugin-fetch-schema",
+    buildStart() {
+      fetchRemoteSchemas();
+    },
+  };
 }
 
-function sha256(data: string): string {
-    return crypto.createHash('sha256').update(data, 'utf8').digest('hex');
+function sha256( data: string, ): string {
```

Comment:
Those whitespaces are rather uncommon. Let's stay with the prettier default to not have them